### PR TITLE
minor: add human eval

### PR DESCRIPTION
**Pull request by zhyncs**
## Checklist

- [ ] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [ ] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [ ] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

File: test/srt/test_nightly_human_eval.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        cls.eval_process = None
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.process:
+            kill_child_process(cls.process.pid)
+        if cls.eval_process:
+            kill_child_process(cls.eval_process.pid)
+
+    def launch_server(self, model, is_fp8, is_tp2):
```
Comment (merrymercy) [inline test/srt/test_nightly_human_eval.py:42]: try to reuse this from test_nightly_gsm8k_eval.py

**Review by merrymercy — CHANGES_REQUESTED** [review]

File: test/srt/test_nightly_human_eval.py (hunk 1 above)
Comment (zhyncs) [reply test/srt/test_nightly_human_eval.py:42]: Ok. BTW I found that there is a bit of an issue with the current human eval for Llama 3.1 8B and 70B, and I need to take some time to identify the cause.

**Comment by merrymercy** [comment]
@zhyncs I think the human eval scores should be good. Can we merge this?

**Comment by zhyncs** [comment]
Let me update the code

**Comment by zhyncs** [comment]
ref https://github.com/sgl-project/sglang/actions/runs/11625335533

File: test/srt/test_nightly_human_eval.py (hunk 1 above)
Comment (zhyncs) [reply test/srt/test_nightly_human_eval.py:42]: It can be written as a static method, I'll change it in the follow-up.

**Pull request merged**